man/man8/zpool-status.8

```diff
@@ -69,6 +69,8 @@ See the
 option of
 .Nm zpool Cm iostat
 for complete details.
+.It Fl e
```

Comment:
You'll want to add '-e' to the top list of flags:
```diff
- .Op Fl DigLpPstvx
+ .Op Fl eDigLpPstvx
```

Reply:
Thanks @tonyhutter. That's fixed in latest commit